massgen/mcp_tools/hooks.py

```
@@ -821,6 +822,205 @@ async def execute(
+    def _get_timeout_for_current_round(self) -> Optional[int]:
+        """Return timeout based on round number (0 = initial, 1+ = subsequent)."""
+        round_num = self.get_agent_round()
+        if round_num == 0:
+            return self.initial_timeout_seconds
+        else:
+            return self.subsequent_timeout_seconds
+
+    async def execute(
+        self,
+        function_name: str,
+        arguments: str,
```

Ruff (0.14.10)

Unused method argument: `arguments`

(ARG002)